hubmap_api_py_client/internal.py

```diff
@@ -109,3 +117,7 @@ def _post_and_get_results(self, url, request_dict):
     def _post_and_get_handle(self, url, request_dict):
         response = self._post_and_get_results(url, request_dict)
         return response[0][HANDLE]
+
+    def _post_and_get_value(self, url, request_dict):
+        response = self._post_and_get_results(url, request_dict)
+        return response['maximum_value']
```

Comment:
- This goes back to the API code, but it would be good not to proliferate response formats. If there were an error, like I pass a bad value in the `request_dict`, how would it show up? Think it would be better to consistently return either `results` or `message`.
- Could this change to bounds, and have it return both the max and the min? It seems asymmetrical to only be able to get one, but not the other.

Reply:
I do like that idea for symmetry purposes.  I would point out that the lower bounds for all three data types are going to be ~ 0.  However since we only encode the sparse version of RNA (and soon ATAC), coming up with a solution that's guaranteed correct and sufficiently performant will take a little bit of brainstorming.